Make it clear you can only delete issues, not events (#2555)
getsentry/sentry-docs: src/platforms/common/data-management/sensitive-data/index.mdx

If you’ve accidentally sent sensitive data to the server it’s likely you’re not going to want to leave it there. There are a few things to note in removal:

- If you send it as a tagged value, removing the event is not enough. You can visit Project Settings and under Tags you’ll find a way to permanently remove any related data for a given tag.

> - You cannot delete single events, but you can delete the issue the event is in by clicking on the trash icon on the issue details page.

Reviewer: ```suggestion
-  While you cannot delete a single event, you can delete the issue the event is in by clicking the trash icon on the issue details page.
```
